source/CardGames.py

```diff
@@ -13,6 +13,37 @@ def find_root_dir():
     cwd = os.path.join( cwd, '..')
   return cwd
 
+def initializeGame():
+  deck = Deck()
+  dealer = Dealer(deck)
+  pot = Pot(0)
+  playerNames = []
+  morePlayers = True
+  buyIn = int(input("Enter buy-in ammount: "))
+  while morePlayers:
+    anotherPlayer = input("Add another player? (Y/N): ")
+    if anotherPlayer == "Y" or anotherPlayer == "y":
+      newPlayer = input("Enter name of new player: ")
+      playerNames.append(newPlayer)
+    else:
+      morePlayers = False
+  pot.addPot(buyIn * len(playerNames))
+  pot.printPot()
+  initPlayers = [] 
+  for p in playerNames:
+    initPlayers.append(Player(p))
+  dealer.dealCards(2, initPlayers)
+
+  return dealer, initPlayers, pot
+
```

Comment:
Something here isnt working right. You shouldnt be adding the entire initialize game function since it already exists in dev_0. It should instead show that you are editing the existing function

Reply:
Yeah I agree with Roy in that we shouldn't have a duplicate function unless I'm not seeing something.